extensions/SharkPool/BetterInput.js

```diff
@@ -0,0 +1,1305 @@
+            },
+          },
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Effects",
+          },
+          {
+            opcode: "setEnterEffect",
+            blockType: Scratch.BlockType.COMMAND,
+            text: "set enter effect to [ENTER_EFFECT]",
+            blockIconURI: effectIcon,
+            arguments: {
+              ENTER_EFFECT: {
+                type: Scratch.ArgumentType.STRING,
+                menu: "enterEffectMenu",
+                defaultValue: "None",
+              },
+            },
+          },
+          {
+            opcode: "setEnterSpeed",
+            blockType: Scratch.BlockType.COMMAND,
+            text: "set enter effect speed to [SPEED]",
+            blockIconURI: effectIcon,
+            arguments: {
+              SPEED: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "10",
+              },
+            },
+          },
+          {
+            opcode: "setExitEffect",
+            blockType: Scratch.BlockType.COMMAND,
+            text: "set exit effect to [EXIT_EFFECT]",
+            blockIconURI: effectIcon,
+            arguments: {
+              EXIT_EFFECT: {
+                type: Scratch.ArgumentType.STRING,
+                menu: "exitEffectMenu",
+                defaultValue: "None",
+              },
+            },
+          },
+          {
+            opcode: "setExitSpeed",
+            blockType: Scratch.BlockType.COMMAND,
+            text: "set exit effect speed to [SPEED]",
+            blockIconURI: effectIcon,
+            arguments: {
+              SPEED: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "10",
+              },
+            },
+          },
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Operations",
+          },
+          {
+            opcode: "setSubmitEvent",
+            blockType: Scratch.BlockType.COMMAND,
+            text: "set force input to [ENTER]",
+            arguments: {
+              ENTER: {
+                type: Scratch.ArgumentType.STRING,
+                menu: "enterMenu",
+                defaultValue: "Disabled",
+              },
+            },
+          },
+          {
+            opcode: "setMaxBoxCount",
+            blockType: Scratch.BlockType.COMMAND,
+            text: "set max box count to: [MAX]",
+            arguments: {
+              MAX: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: 1,
+              },
+            },
+          },
+          {
+            opcode: "getBoxCount",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "box count",
+          },
+          {
+            opcode: "getMaxCount",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "box limit",
+          },
+        ],
+        menus: {
+          alignmentMenu: ["left", "right", "center"],
+          fontMenu: [
```

Comment:
A vast majority of these fonts will not work for a *lot* of our users (anyone on ChromeOS, macOS, Linux)

Do what animated text does - stick to the standard scratch fonts. When custom fonts are merge then you can use those too. As long as it's acceptReporters: true, people will be able to manually drop in whatever they want anyways

Reply:
I can remove the custom fonts stuff. Ill also try to have accept reporters in the next update whenever it comes. If you are merging this file right now, are you modifying that stuff or am I?

Reply:
Saving this for last...

Reply:
You can do custom fonts now. See animated text